src/main/utils/contextUtils.ts

```diff
@@ -0,0 +1,37 @@
+import * as fse from 'fs-extra';
+import * as path from 'path';
+import md5 from 'md5';
+import * as os from 'os';
+import { TreesManager } from '../treeDataProviders/treesManager';
+
+export class ContextUtils {
```

Comment:
The functions here doesn't deserved to be under `ContextUtils` because there is no Context here. I'm afraid we don't need this class anymore.

Reply:
Removed